.github/scripts/docs.go

```diff
@@ -0,0 +1,22 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package main
+
+import (
+	"os"
+
+	"github.com/dapr/cli/cmd"
+	"github.com/spf13/cobra/doc"
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		panic("Requires a path to generate docs in.")
+	}
+	// Tag contains date. Need to disable generation to avoid date generation errors.
+	cmd.RootCmd.DisableAutoGenTag = true
```

Comment:
Example of error: https://github.com/dapr/cli/pull/446/checks?check_run_id=1010866665#step:8:89